creator.html

```diff
@@ -49,8 +44,8 @@ <h3 class="h3 text-center mb-5">About Kodular Creator</h3>
                         </div>
                         <div class="col-10">
                             <h5 class="feature-title">No Coding!</h5>
-                            <p class="grey-text">Thanks to Kodular Creator, you can create any app without programming a single line of code.
-                                Just drag and drop a few components, join some blocks and you will have your desired app done</p>
+                            <p class="grey-text">Thanks to Kodular Creator, you can create any app without writing a single line of code.
+                                Just drag n' drop the required components, join some blocks and voila! You just created your awesome app.  </p>
```

Comment:
```suggestion
                                Just drag n' drop a few components, join some blocks and voila! You have just created your awesome app.  </p>
```

1. Saying few is better than required. New users can think it's too complex if you say required instead of few.
2. Little grammar mistake